### PR TITLE
Delete mds-providers package and migrate repo to mds-provider-service

### PR DESCRIPTION
**Pull request by janedotx**
## 📚 Purpose
Delete mds-providers package and migrate repo to mds-provider-service

## 👌 Resolves:
- [ ] 🐛 JRA-834 fixes bug
- [ ] ✨ JRA-756 implements new feature

## 📦 Impacts:
*[list of packages]*

## ✅ PR Checklist
- [ ] Add or remove checklist items to suit your needs

### CODE REVIEW

File: packages/mds-compliance-engine/tests/engine/engine-test.spec.ts [hunk 1]
```diff
@@ -146,9 +153,11 @@ describe('Verifies compliance engine processes by vehicle most recent event', ()
     }, []) as VehicleEventWithTelemetry[]
     await db.seed({ devices, events, telemetry: events.map(({ telemetry }) => telemetry) })
     const inputs = await getAllInputs()
+    console.log('inputz')
+    console.log(JSON.stringify(inputs))
```
Comment (avatarneil) [inline packages/mds-compliance-engine/tests/engine/engine-test.spec.ts:157]: Should delete these guys

**Review by avatarneil — APPROVED** [review]
LGTM pending deletion of the console.log statements! 

One thing to note, we'll need to make sure that deployments for the next release are all updated to include the new provider-service.

File: packages/mds-compliance-batch-processor/batch_process.ts [hunk 2]
```diff
@@ -45,7 +45,7 @@ export async function computeSnapshot() {
   })
   const compliance_as_of = now() // The timestamp right after we fetch the inputs (latest state as of that time)
 
-  const provider_ids = Object.keys(await getProviders())
+  const provider_ids = (await ProviderServiceClient.getProviders()).map(p => p.provider_id)
```
Comment (drtyh2o) [inline packages/mds-compliance-batch-processor/batch_process.ts:48]: This appears often enough that I wonder if it warrants its own service method. NBD.

**Review by drtyh2o — COMMENTED** [review]
What do downstream consumers need to do now that this package is gone? Stick with the prior version until upgraded? Is that another PR?

**Review by nessur — APPROVED** [review]

**Comment by janedotx** [comment]
> What do downstream consumers need to do now that this package is gone? Stick with the prior version until upgraded? Is that another PR?

Do you mean consumers in lacuna-core? Yeah, that's going to be another PR.

**Pull request merged**